org.adempiere.base.process/src/org/compiere/process/ImportProduct.java

```diff
@@ -559,10 +562,12 @@ protected String doIt() throws java.lang.Exception
 						.append("SET (Value,Name,Description,DocumentNote,Help,")
 						.append("UPC,SKU,C_UOM_ID,M_Product_Category_ID,Classification,ProductType,")
 						.append("Volume,Weight,ShelfWidth,ShelfHeight,ShelfDepth,UnitsPerPallet,")
+						.append("CustomsTariffNumber,Group1,Group2,")

```

Comment:
This is just the "update" case, you must take care also in the "insert" case in line 541:
`MProduct product = new MProduct(imp);`
The constructor MProduct(X_I_Product) must also assign these three new columns.

Also - as you added columns, please generate the classes X_/I_ for M_Product and I_Product.

Reply:
Done. To do the first model classes where required anyway.